alicloud/resource_alicloud_log_alert.go

```diff
@@ -404,21 +416,24 @@ func getNotiMap(v *sls.Notification) map[string]interface{} {
 	mapping := make(map[string]interface{})
 
 	mapping["content"] = v.Content
-	if v.Type == "SMS" {
-		mapping["type"] = "SMS"
+	if v.Type == sls.NotificationTypeSMS {
+		mapping["type"] = sls.NotificationTypeSMS
 		mapping["mobile_list"] = v.MobileList
 	}
 
-	if v.Type == "Email" {
-		mapping["type"] = "Email"
+	if v.Type == sls.NotificationTypeEmail {
+		mapping["type"] = sls.NotificationTypeEmail
 		mapping["email_list"] = v.EmailList
 	}
 
-	if v.Type == "DingTalk" {
-		mapping["type"] = "DingTalk"
+	if v.Type == sls.NotificationTypeDingTalk {
+		mapping["type"] = sls.NotificationTypeDingTalk
 		mapping["service_uri"] = v.ServiceUri
 	}
 
+	if v.Type == sls.NotificationTypeMessageCenter {
+		mapping["type"] = sls.NotificationTypeMessageCenter
+	}
 	return mapping
 
 }
```

Comment:
Thanks for your contribution. Please update it in the docs.

Reply:
Done, PTAL